### DISCUSSION TITLE
HAA devices filter?

### DISCUSSION THREAD

**Discussion by juansf99** [RavenSystem/esp-homekit-devices#1566] [HAA Home App]
Hi,
If you have many devices and only a few with HAA, it is difficult to find them.
I think it would be interesting an option to filter only the devices with HAA and leave out the others in the "devices" tab since I will not be able to update the firmware of these from here.
Maybe a switch in the settings?

**Comment by RavenSystem** [comment]
I will add a new tab with HAA devices only.
